hack/test/e2e-integration.sh

```diff
@@ -85,8 +86,7 @@ e2e_run "timeout=\$((\$(date +%s) + ${TIMEOUT}))
 		 done"
 
 ## Run conformance tests if var is not null
-if [ -n "${CONFORMANCE}" ]
-then
+if [ ${CONFORMANCE:-"dontrun"} == "run" ]; then
```

Comment:
-n conformance fails with our options above ( set -eou pipefail ), so we need to ensure it has some default value so they can be properly skipped